fw/http_sess.c

```diff
@@ -799,40 +526,21 @@ tfw_http_sticky_req_process(TfwHttpReq *req, StickyVal *sv, TfwStr *cookie_val)
 	r = tfw_http_sticky_get_req(req, cookie_val);
```

Comment:
With the check `jiffies > sv->ts + (unsigned long)sticky->sess_lifetime * HZ` in `tfw_http_sticky_verify()` we have the sticky cookie life time, i.e. we should set appropriate `Max-Age` cookie parameter equal to this limit. This way we'll avoid flood of `sticky cookie value expired` messages. We also may add something like "Please make sure that you have appropriate Max-Age for your sticky cookie configuration" to the message to provide some help to an administrator.

Reply:
Currently administrator can specify Max-Age in cookie options. Should we specify it if administrator don't do it? Should we parse options to find Max-Age directive?